imx/metadata.go

```diff
@@ -21,7 +21,7 @@ func (c *Client) AddMetadataSchemaToCollection(
 	contractAddress string,
 	addMetadataSchemaToCollectionRequest api.AddMetadataSchemaToCollectionRequest,
 ) (*api.SuccessResponse, error) {
-	timestamp, signature, err := getProjectOwnerAuthorisationHeaders(l1signer)
+	timestamp, signature, err := createIMXAuthorisationHeaders(l1signer)
```

Comment:
I believe this is now `generateIMXAuthorisationHeaders` after the TS Core SDK changes

Reply:
It is a internal function, so naming is not required to be consistent.

Reply:
correct but it follows the same pattern now and even internally, consistency is important for contributors.

Reply:
Not sure if I agree, but updated this time.